designs/design-template.stanza

```diff
@@ -0,0 +1,49 @@
+#use-added-syntax(jitx)
+defpackage design-template :
+  import core
+  import jitx
+  import jitx/commands
+
+; =========================
+; Module to run as a design
+; ==========================
+val board-shape = RoundedRectangle(30.0, 18.5, 0.25)
```

Comment:
Normally in idiomatic stanza we would notate global constants as `BOARD-SHAPE`, do we want to keep the same style here?

Reply:
Sorry, global in the sense that it is defined at the top level of a package. It may be preferable to move it to the body of `run-design` if it's intended to be local

Reply:
Usability wise, it helps to have the shape next to the definition of the `pcb-module` that will go in that design. I tried it inside the body of `run-design` but did not like the resulting legibility.